### ISSUE TITLE
Question about train code

### ISSUE THREAD

**Issue by terry2100** [wusize/OpenUni#5]
Thanks  for your work!
In the code for image2image_loss, why  set vit_embeds.requires_grad = True?

**Comment by BecauseImBatman0** [comment]
+1 , and why "left padding for editing" in collate_func_img2img() ?
